### ISSUE TITLE
marker popup ugly scrollbars

### ISSUE THREAD

**Issue by harry-wood** [maproulette/maproulette2#283]
These ugly scrollbars don't look right at all.

![screen shot 2017-03-03 at 23 39 01](https://cloud.githubusercontent.com/assets/227525/23573042/3570df9e-006b-11e7-9355-b166c25ebc8c.png)

This doesn't happen on some of the markers where the description text is much shorter.

**Comment by mvexel** [comment]
Agreed the layout for that is pretty ugly overall. Needs work! I'd be happy to take a PR -- should not be too much work to make this look much nicer!!

**Comment by mvexel** [comment]
Related: #227
